### DISCUSSION TITLE
loader POST method ELK Help

### DISCUSSION THREAD

**Discussion by GoBenGoh** [mulesoft-catalyst/metrics-toolkit#161] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76719511/143359753-877abbda-a29e-424f-8fcb-bff7ae8d68b8.png)
Why is the toolkit trying to HTTP POST on http://https://url when the url should be https://url ?

**Comment by richardmckinley** [comment, marked as answer]
Hi @GoBenGoh, 

It looks like the HTTP request within the ELK loader is configured with HTTP only, which is why it does a HTTP POST rather than HTTPS. 
![image](https://user-images.githubusercontent.com/58299919/143414887-14841e5e-c020-477a-b23e-1b6b78417a19.png)
You can reconfigure this HTTP Requester to use HTTPS if required. The toolkit is designed to be easily extensible. 

Also, the host parameter should just contain the host name, not the protocol. 

If you think we should make this dynamic, similar to how it is done for the Splunk loader, please feel free to ask for an enhancement by raising an issue.
